yt/geometry/particle_geometry_handler.py

```diff
@@ -6,12 +6,12 @@
 from glob import glob
 
 import numpy as np
+from ewah_bool_utils.ewah_bool_wrap import BoolArrayCollection
```

Comment:
It seems worth noting that this is the one and only import that makes ewah_bool_array a _runtime_ dependency. If we could somehow avoid it, it would only be a dependency at build time, which would be much more robust to potential ABI compat breakage.

Reply:
Hm, is that true?  I think that even though we're building the cython files, when we do the `cimport` it will be using `dlopen` and make it a runtime dependency as well.